tutorials/editor/project_settings.rst

```diff
@@ -1,16 +1,138 @@
-:article_outdated: True
-
 .. _doc_project_settings:
 
 Project Settings
 ================
 
-This page explains how to use the Project Settings window. If you would like to access and modify project settings via code, see :ref:`ProjectSettings <class_ProjectSettings>`.
+There are dozens of settings you can change to control a project's execution,
+including physics, rendering, and windowing settings. These settings can be
+changed from the **Project Settings** window, from code, or by manually editing
+the ``project.godot`` file. You can see a full list of settings in the 
+:ref:`ProjectSettings <class_ProjectSettings>` class.
+
+Internally, Godot stores the settings for a project in a ``project.godot`` file,
+a plain text file in INI format. While this is human-readable and version control
+friendly, it's not the most convenient to edit. For that reason, the
+**Project Settings** window is available to edit these settings. To open the
+Project Settings, select **Project > Project Settings** from the main menu.
+
+.. figure:: img/project_settings_basic.webp
+    :align: center
+
+    The Project Settings window
+
+The **Project Settings** window is mainly used to change settings in the
+**General** tab. Additionally, there are tabs for the
+:ref:`Input Map <doc_input_examples_input_map>`,
+:ref:`Localization <doc_internationalizing_games>`,
+:ref:`Globals <doc_singletons_autoload>`,
+:ref:`Plugins <doc_installing_plugins_enabling_a_plugin>`, and
+**Import Defaults**. Usage of these other tabs is documented elsewhere.
+
+Changing project settings
+-------------------------
+
+The **General** tab of the project settings window works much like the inspector.
+It displays a list of project settings which you can change, just like inspector
+properties. There is a list of categories on the left, which you can use to select
+related groups of settings. You can also search for a specific setting with the
+**Filter Settings** field.
+
+Each setting has a default value. Settings can be reset to their default values
+by clicking the circular arrow **Reset** button next to each property.
+
+Changing project settings from code
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can use :ref:`set_setting() <class_ProjectSettings_method_set_setting>` to
+change a setting's value from code:
+
+.. tabs::
+    .. code-tab:: gdscript GDScript
+        
+        ProjectSettings.set_setting("application/run/max_fps", 60)
+        ProjectSettings.set_setting("display/window/size/mode", DisplayServer.WINDOW_MODE_WINDOWED)
```

Comment:
For the set of example project settings used throughout the page, I picked these two because:
- I know them reasonably well
- They both need runtime duplicates, in different classes
- They have different types (int and enum) which is mildly instructive especially for the C# examples
- They are reasonably common

I'm open to suggestions for alternate project settings to use as examples. We can also add a third example if there is a good *string* example to use.